pallets/detach/src/lib.rs

```diff
@@ -161,7 +161,7 @@ pub mod pallet {
 
 	#[pallet::storage]
 	pub type DetachNonces<T: Config> =
-		StorageDoubleMap<_, Blake2_128Concat, Vec<u8>, Blake2_128Concat, SupportedChains, u64>;
+		StorageDoubleMap<_, Twox64Concat, Vec<u8>, Twox64Concat, SupportedChains, u64>;
```

Comment:
Why did you change `Blake2_128Concat` to `Twox64Concat` btw? @sinkingsugar

Reply:
Smaller footprint. Enough security.